lib/data/article/service/local/model/article_db_model.dart

```diff
@@ -35,6 +35,7 @@ class ArticleDbModel extends HiveObject {
 
   Article toArticle() {
     return Article(
+      id: null,
```

Comment:
Just out of curiosity, why is this explicitly declared as null instead of just not being added?

Reply:
I expected articles would have an id, then realised the api behind this template didn't have one 🤷‍♂️